src/main/deploy/swerve/modules/backleft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "location": {
+    "front": 0,
+    "left": 0
+  },
+  "absoluteEncoderOffset": 0,
```

Comment:
This needs to be filled in. We can't know the `absoluteEncoderOffset` yet (we'll have to measure it), but we do know the frame dimensions so we can calculate the front/left positions.